src/psioptionseditor.cpp

```diff
@@ -135,11 +135,18 @@ PsiOptionsEditor::PsiOptionsEditor(QWidget *parent) : QWidget(parent)
     tpm_ = new QSortFilterProxyModel(this);
     tpm_->setSourceModel(tm_);
 
-    QVBoxLayout *layout = new QVBoxLayout(this);
-    layout->setContentsMargins(0, 0, 0, 0);
+    QVBoxLayout *mainLayout = new QVBoxLayout(this);
+    mainLayout->setContentsMargins(10, 10, 10, 10); // Adjust margins for the main layout
+
+    QFrame *frame = new QFrame(this);
```

Comment:
what is this frame for? please attach screenshots

Reply:
This creates a new widget that is a child of the current widget (this). The QFrame can be used to contain other widgets and apply a frame around them, which can be useful for grouping related UI elements together visually.
![Screenshot (1)](https://github.com/psi-im/psi/assets/115822066/5a17b6c6-ebab-49c8-913a-ab5c62b5f49a)

Reply:
This creates a new  widget that is a child of the current widget (this). The QFrame can be used to contain other widgets and apply a frame around them, which can be useful for grouping related UI elements together visually.
![308119581_497084679136218_7468071683868194394_n](https://github.com/psi-im/psi/assets/115822066/f0556868-d3d4-4906-b15c-6dd38ef71ab7)

Reply:
This creates a new widget that is a child of the current widget (this). The QFrame can be used to contain other widgets and apply a frame around them, which can be useful for grouping related UI elements together visually.
![Screenshot (246)](https://github.com/psi-im/psi/assets/115822066/4fdac000-128a-4b89-9a1c-70f8a57278fd)